### ISSUE TITLE
Demo has a communication error

### ISSUE THREAD

**Issue by psavery** [ImagingDataCommons/slim#187]
The demo here appears to have a communication error: https://imagingdatacommons.github.io/slim/

I believe it was working a few weeks ago.

**Comment by fedorov** [comment]
Yes, we are aware of this regression, and are looking into resolving this.

@wlongabaugh this is due to the issue we discussed in the email thread with @hackermd. This was also reported by @sedghi earlier this week.

**Comment by fedorov** [comment]
We are looking into how to re-enable that demo, but it is difficult to give any promises on when this might happen, at this point.

I know it is not sufficient to cover all samples we had in the demo server, but here are some public samples from [Imaging Data Commons](https://portal.imaging.datacommons.cancer.gov/) to demonstrate at least some aspects of Slim.

* H&E: https://viewer.imaging.datacommons.cancer.gov/slim/studies/2.25.211094631316408413440371843585977094852/series/1.3.6.1.4.1.5962.99.1.208792987.352384958.1640886332827.2.0
* multichannel fluorescence: https://viewer.imaging.datacommons.cancer.gov/slim/studies/2.25.93749216439228361118017742627453453196/series/1.3.6.1.4.1.5962.99.1.2344794501.795090168.1655907236229.4.0?state=1.2.826.0.1.3680043.10.511.3.79630386778396943986328353882008803

If you are interested in other specific aspects of functionality, please comment!

**Comment by Rikaelus** [comment]
@fedorov Could you guys add these links to the readme, at least temporarily? These have helped to get a little further in my Slim research that's been pretty slow-going.

**Comment by fedorov** [comment]
Good point - done!
